code/obj/item/misc_weapons.dm

```diff
@@ -538,6 +538,28 @@
 		..()
 		setProperty("movespeed", -0.5)
 
+//chaplain traitor knice
+/obj/item/dagger/syndicate/chaplain_knife
+	name = "Blade of Light"
+	desc = "A knife made out of pure light. Now thats what I call blind faith"
+	item_state = "chap_dagger"
+	force = 14
+	throwforce = 18
+	stamina_cost = 5
+	c_flags = EQUIPPED_WHILE_HELD
+
+	New()
+		..()
+		setProperty("movespeed", -0.4)
```

Comment:
Why does this adjust move speed?

Reply:
its a carry over from the syndicate infiltrator dagger, though nerfed comparatively. It helps give it utility and flavor, although if you believe it would be too powerful, it can be either nerfed or outright removed.